src/plugins/intel_gna/gna_plugin_config.cpp

```diff
@@ -276,7 +276,7 @@ OPENVINO_SUPPRESS_DEPRECATED_START
             }
 OPENVINO_SUPPRESS_DEPRECATED_END
         } else if (key == CONFIG_KEY(LOG_LEVEL) || key == ov::log::level) {
-            if (value == PluginConfigParams::LOG_WARNING || value == PluginConfigParams::LOG_NONE) {
+            if (value == PluginConfigParams::LOG_WARNING || value == PluginConfigParams::LOG_NONE || value == PluginConfigParams::LOG_DEBUG) {
                 gnaFlags.log_level = ov::util::from_string(value, ov::log::level);
             } else {
                 log << "Currently only LOG_LEVEL = LOG_WARNING and LOG_NONE are supported, not " << value;
```

Comment:
Comment should be updated here as well. It's worth considering if not all levels should be enabled at this point. Some could just have no real effect, but would be available for use.

Reply:
Done